packages/app-root/src/hooks/useAdminMode.js

```diff
@@ -7,17 +7,21 @@ const adminBorderImage = 'repeating-linear-gradient(45deg,#000,#000 25px,#ff0 25
 
 export default function useAdminMode(user) {
   const [adminState, setAdminState] = useState(storedAdminFlag)
-  const adminMode = user?.admin && adminState
+  const userIsLoaded = user !== undefined
```

Comment:
The project app has `user.loadingStatus` but that doesn't exist in the root app, so I'm assuming that the user is undefined while loading, then null if loaded but not logged in.